Library/Invoice.cs

```diff
@@ -395,6 +395,26 @@ public Invoice RefundAmount(int amountInCents, RefundMethod method = RefundMetho
                 return null;
         }
 
+        /// <summary>
+        /// Enter an offline payment for a manual invoice
+        /// </summary>
+        /// <param name="transaction">The transaction to be entered.</param>
+        /// <returns>new Transaction object</returns>
+        public Transaction EnterOfflinePayment(Transaction transaction)
+        {
+            var successfulTransaction = new Transaction();
+
+            var response = Client.Instance.PerformRequest(Client.HttpRequestMethod.Post,
```

Comment:
Kind of a nit, but might make more sense to call this `statusCode`. Makes line 412 a little confusing.

Reply:
I copied and pasted from this
https://github.com/recurly/recurly-client-net/blob/38c67ce96637c083aa1c5edd49b2a1ffbbd777c2/Library/Invoice.cs#L293
Do you want me to go ahead and change it to `statusCode` everywhere in the file? I'd rather it be consistent if I change mine.

Reply:
Yeah, I think it makes more sense if it's easy to change.

Reply:
Fixed